### PR TITLE
Night mode

### PR DESCRIPTION
**Pull request by naveed-ahmad**
* Added night mode
* Optimized Surah name svg( removed comments and fill colors etc, added class name to style svg using css)
* Minor css fixes

### CODE REVIEW

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32994

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32996

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32997

**Comment by sabeurthabti** [comment]
maybe add the night mode option within the burger menu?

**Comment by naveed-ahmad** [comment]
@sabeurthabti disagree sorry, having setting in three places(burger menu/sidebar, Surah top, and along with player)  will complicate things, isn't good usability. Users are already asking about how to repeat single ayah, https://github.com/quran/quran.com-frontend/issues/478, https://github.com/quran/quran.com-frontend/issues/476, https://github.com/quran/quran.com-frontend/issues/457, https://github.com/quran/quran.com-frontend/issues/456

Ill suggest to make player bar full with and add setting there, everything. On mobile screen add setting icon ( cog ) clicking it show popup for settings. So next PR maybe for fixing the settings.

Already working on mockups, will share when they are ready before implementing.

File: src/components/NightModeToggle/index.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+  static defaultProps = {
+    isNightMode: false
+  };
+
+  toggleNightMode = () => {
+    const { isNightMode } = this.props;
+
+    if(isNightMode)
+      document.body.classList.add('night-mode');
```
Comment (sabeurthabti) [inline src/components/NightModeToggle/index.js]: better alternative `document.body.classList.toggle('night-mode');`

Comment (sabeurthabti) [inline src/components/NightModeToggle/index.js]: can we also make sure to use brackets for if statements 
``` javascript 
if (x) {
...
}
```

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:33001

**Comment by sabeurthabti** [comment]
@naveed-ahmad amazing stuff as always

File: src/components/NightModeToggle/index.js (hunk 1 above)
Comment (naveed-ahmad) [inline src/components/NightModeToggle/index.js]: I always thought `toggle` is a jquery function, thanks for good suggestion. Also brackets aren't required for single statement, but if we are following some conventions happy to follow them.

**Comment by naveed-ahmad** [comment]
Thanks @sabeurthabti

**Pull request merged**